cypress/integration/dataset_details_page_spec.js

```diff
@@ -173,12 +173,6 @@ describe('Write SQL Tab for System dataset', function () {
     cy.get(savePopover).should('be.visible')
     cy.get(saveButton).should('be.disabled')
     cy.get(queryPrompt).type("My Query")
-    cy.get(saveButton).not('be.disabled')
```

Comment:
👍 For keeping these up to date

Reply:
Agreed. Do you have another test for the popup functionality?

Reply:
We no longer support a user even *trying* to save a visualization if they are not authenticated.  The save button will never become enabled because the user can't log in for this test.  Thus we just omitted these steps.